config/php/docker-entrypoint.sh

```diff
@@ -62,148 +18,37 @@ function waitUntil() {
-    local port=15672
-    local user=$(echo $1 | php -r "echo @parse_url(stream_get_contents(STDIN))['user'];")
-    local pass=$(echo $1 | php -r "echo @parse_url(stream_get_contents(STDIN))['pass'];")
-    local path=$(echo $1 | php -r "echo @parse_url(stream_get_contents(STDIN))['path'];")
-    local vhost=$(echo $1 | php -r "echo (@explode('/', @parse_url(stream_get_contents(STDIN))['path']))[1];")
-
-    local  url="http://${host}:${port}/api/vhosts/${vhost}"
-    local  userPass="${user}:${pass}"
-    curl -u "${userPass}" --fail "${url}" ||  curl  --silent  --show-error  -u "${userPass}" --fail -X PUT "${url}"
-}
-
 # first arg is `-f` or `--some-option`
 if [ "${1#-}" != "$1" ]; then
 	set -- php-fpm "$@"
 fi
 
+>&2 echo "Create directories"
+mkdir -p var/cache var/log public/multimedia public/thumbnail public/avatar import export
+chown -R www-data var public/multimedia public/thumbnail public/avatar import export
 
-envsubst < /usr/local/etc/php/php-ini-directives.ini.template | sed "s~;\s*~\n~g"  > /usr/local/etc/php/conf.d/php-ini-directives.ini
-
-if [ "$1" = 'php-fpm' ] || [[ "$1" =~ (vendor/)?bin/.* ]] || [ "$1" = 'composer' ]; then
-
-	PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-production"
-	if [ "$APP_ENV" != 'prod' ]; then
-		PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-development"
-	fi
-	echo "Linking ${PHP_INI_RECOMMENDED} > ${PHP_INI_DIR}/php.ini"
-	ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini"
-
-    disableXdebug
-    genereJwtKeysIfInvalid "${JWT_PRIVATE_KEY_PATH}" "${JWT_PUBLIC_KEY_PATH}" env:JWT_PASSPHRASE
-fi
-
-if [[ "$1" =~ bin/console ]] && [[ "$2" = 'messenger:consume' ]]; then
-
-    createAmqpVhost "${MESSENGER_TRANSPORT_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_IMPORT_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_CORE_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_EXPORT_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_DOMAIN_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_CHANNEL_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_SEGMENT_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_COMPLETENESS_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_NOTIFICATION_DSN}"
-
-    bin/console messenger:setup-transports --no-interaction import
-    bin/console messenger:setup-transports --no-interaction channel
-    bin/console messenger:setup-transports --no-interaction export
-    bin/console messenger:setup-transports --no-interaction core
-    bin/console messenger:setup-transports --no-interaction event
-    bin/console messenger:setup-transports --no-interaction segment
-    bin/console messenger:setup-transports --no-interaction completeness
-    bin/console messenger:setup-transports --no-interaction notification
+>&2 echo "Generate JWT keys"
+genereJwtKeys "config/jwt/private.pem" "config/jwt/public.pem"
 
-    >&2 echo "messenger initialization finished"
-fi
+>&2 echo "Composer install"
+#composer install --prefer-dist --no-progress --no-suggest --no-interaction
```

Comment:
I guess it could stay. Otherwise will have to be done manually

Reply:
It will be uncomment when i finish :)